pages/common/gt.md

```diff
@@ -0,0 +1,32 @@
+# gt
+
+> Create and manage sequences of dependent code changes (stacks) for `git` and GitHub.
+> More information: <https://docs.graphite.dev/>.
+
+- Authenticate the CLI with GitHub:
+
+`gt auth --token {{your_cli_auth_token}}`
```

Comment:
Is this the PAT from GitHub? We try to avoid `your`. Maybe this is more clear
```suggestion
`gt auth --token {{github_pat}}`
```

Reply:
No, I believe this is an auth token from `graphite.dev` and then their API performs the GitHub actions on your behalf; I shall rename the variable a bit to make this more obvious